core/domain/exp_services.py

```diff
@@ -1982,6 +1983,36 @@ def create_or_update_draft(
     exp_user_data.put()
 
 
+def try_upgrading_draft_to_exp_version(exp_id, user_id):
+    """Try upgrading draft change list to match the latest exploration version.
+
+    If successful (all changes successfully upgraded), we commit the upgraded
+    draft change list to the ExplorationuserDataModel and increment
```

Comment:
ExplorationuserDataModel --> ExplorationUserDataModel

Reply:
Done, thanks for the catch!